api/envoy/protobuf/BUILD

```diff
@@ -0,0 +1,7 @@
+# DO NOT EDIT. This file is generated by tools/proto_sync.py.
```

Comment:
@htuch: I created this file by hand. I'm guessing this was the wrong thing to do?

Reply:
Correct